test/Microsoft.AspNet.Mvc.DataAnnotations.Test/DataAnnotationsModelValidatorTest.cs

```diff
@@ -49,6 +54,8 @@ public static IEnumerable<object[]> Validate_SetsMemberName_OnValidationContext_
             }
         }
 
+        private const string LocalizationKey = "LocalizeIt";
```

Comment:
Move this down above `Validate_AttributesIncludeValues` or (better) remove entirely.